### PR TITLE
chore: use eslint-config-cozy-app

### PR DESCRIPTION
**Pull request by drazik**
This PR replace our custom ESLint config with eslint-config-cozy-app. I also updated eslint dependency and fixed all the errors the new config thrown.

For the review to be easier, I splitted the errors fixing part in 1 file === 1 commit.

For some errors, I added a `eslint-disable` or `eslint-disable-next-line` comment. I'm pretty sure some of theses lines can be deleted (mostly `console.log` that were surely used for debug purposes). But since I didn't write them, I can't be 100% sure. If you know if a line can be safely removed, please tell me !

### CODE REVIEW

**Comment by ptbrowne** [comment]
If it's only linting, LGTM :)

File: .eslintrc [hunk 1]
```diff
@@ -1,11 +1,6 @@
 {
-  "env": { "es6": true, "jasmine": true },
-  "parser": "babel-eslint",
-  "extends": ["standard", "standard-jsx"],
-  "rules": {
-    "react/jsx-wrap-multilines": 2,
-    "react/react-in-jsx-scope": 2
-  },
+  "env": { "jasmine": true },
```
[the remark this answers is not in the document]
Comment (drazik) [reply .eslintrc]: Je ne sais pas, mais c'est le seul environnement qui était dans notre config mais qui n'est pas dans eslint-config-cozy-app, donc je l'ai laissé.

[the remark this answers is not in the document]
Comment (ptbrowne) [reply .eslintrc]: s/jasmine/jest et ce sera bon :)

ca permet d'avoir les globals describe/it/test etc...

[the remark this answers is not in the document]
Comment (kosssi) [reply .eslintrc]: du coup on peut carrément tout supprimer vu que jest est déjà dans [eslint-config-cozy-app](https://github.com/CPatchane/create-cozy-app/blob/master/packages/eslint-config-cozy-app/index.js#L15)

**Pull request merged**